KLSadd_insertion/README.md

```diff
@@ -15,5 +15,110 @@ NOTE: when you run updateChapters.py or linetest.py you need the KLSadd.tex file
+
+ - updateChapters.py: this is our code. This document will be going over the variables and methods in this program. This program should: take paragraphs from every section in KLSadd and insert them into the relevant chapter file and into the relevant section within that chapter.
+
+ - linetest.py: this abomination of code is the original program. It did the job, and it did it well. However, I realized that it was very very unreadable. This program is the child I never wanted. This program does 100% work and its outputs can be used to check against updateChapters.py's output.
+
+ - newtempchap09.tex and newtempchap14.tex these are output files of linetest.py and this is what the output to updateChapters.py should be like!!!!!!!
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next: the specifications
+
+Toward the end of every section in the chapter file, there is a subsection called "References". It basically just contains a bunch of references to sources and other papers. Every paragraph in every subsection in KLSadd.tex must be put *before* the "References" subsection in the corresponding section in the chapter file. For example:
+**These aren't real sections, obviously, just an example**
+
+in chap09.tex:
+```latex
+\section{Dogs}\index{Dog polynomials}
+\subsection*{Hypergeometric dogs}
+blah blah blah
+\subsection*{References}
+\cite{Dogs101}
+```
+
+in KLSadd.tex:
+
+```latex
+\subsection*{9.15  Dogs}
+\paragraph{Beagles}
+The Wilson polynomial $W_n(y;a,b,c,d)$ is symmetric
+in $a,b,c,d$.
+```
+
+So in the chapter 9 file, section 15 "Dogs", before the References subsection, we need to add in the Beagles paragraph. HOWEVER, there are some changes that need to be made before we insert. The easiest change is adding a comment that tells us something was inserted. We just append something like "%RS insertion begin" and "%RS insertion complete" before and after, respectively, the Beagles paragraph when we insert. Another change is adding \large\bf before the paragraph name. This changes the size and format so it looks prettier. So the heading in this example would be:
+
+```latex
+\paragraph{\large\bf Beagles}
+```
+
+**It is important to note that these changes are currently (3/25/16) unimplemented in updateChapters.py and only exist in linetest.py, if you want to see how the output to updateChapters.py should look, look at newtempchap09.tex**
+
+So after updateChapters.py is called, the chapter 9 file should look like this:
+
+```latex
+\section{Dogs}\index{Dog polynomials}
+\subsection*{Hypergeometric dogs}
+blah blah blah
+%RS insertion begin
+\paragraph{\large\bf Beagles}
+The Wilson polynomial $W_n(y;a,b,c,d)$ is symmetric
+in $a,b,c,d$.
+%RS insertion complete
+\subsection*{References}
+\cite{Dogs101}
+```
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Same as this one - I think there's a way to insert a horizontal line in markdown (see [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#horizontal-rule))